sklearn/model_selection/_search.py

```diff
@@ -552,17 +565,39 @@ def _fit(self, X, y, labels, parameter_iterable):
 
         # Computed the (weighted) mean and std for all the candidates
         weights = test_sample_counts if self.iid else None
-        means = np.average(test_scores, axis=1, weights=weights)
-        stds = np.sqrt(np.average((test_scores - means[:, np.newaxis]) ** 2,
-                                  axis=1, weights=weights))
+
+        time = np.array(_, dtype=np.float64).reshape(n_candidates, n_splits)
```

Comment:
No, we don't use `_` as a variable name. In the former code, it means "we don't want to reuse this variable".